release-notes.txt

```diff
@@ -0,0 +1,216 @@
+BISDN Linux
+v4.6.0 Release Notes
+Date: 2022-05-04
+
+! New in this release
+- New platform Edgecore DCS201 (AS5835-54X) is now supported. The DCS201 is a
+  Trident 3 with 48x 10GbE SFP+ ports and 6x 100GbE QSFP28 ports
+- OF-DPA extensions for ryu and sample code added for configuring MPLS
+  pseudowires
+- OF-DPA: new option for controlling memory allocation for Layer 2 and Layer 3
+  tables
+- baseboxd/OF-DPA: Broadcom KNET interfaces are now fully supported
+- baseboxd updated to version 1.11.2
+- Linux kernel updated to version 5.15.35
+- FRR updated to version 8.2.2
+
+! Bug fixes
+- increased max read buffer to avoid loss of netlink messages (and out of sync
+  switch state) under heavy load
+- AS4610: added missing Kernel option to allow BGP MD5 authentication
+- EPS202 (AS4630-54PE): enabled the LM77 driver, fixing reading out temperatures from
+  the fourth sensor in onlp
+- Broadcom-KNET interfaces: now properly generates random MAC addresses
+- Broadcom-KNET interfaces: fixed issue related to race condition upon
+  interface creation
```

Comment:
```suggestion
- Broadcom-KNET interfaces: fixed a race condition on interface creation that could cause system hangs when configuring interfaces using systemd-networkd
```

(add line breaks as needed)

Reply:
Updated line with the nessesary line break